### PR TITLE
fix: do not try to send notifications when they are disabled

### PR DESCRIPTION
**Pull request by kolaente**
Currently, the command will be triggered from the /health endpoint and will send a notification no matter what is configured. This PR adds a check so that it will only do that when notifications are actually enabled.

In an application I'm building, we don't have mails configured. Because of that, sending the notification fails later on and spams the log with failed attempts, making tracking actual errors way harder.

### CODE REVIEW

File: src/Commands/RunHealthChecksCommand.php [hunk 1]
```diff
@@ -29,7 +29,7 @@ public function handle(): int
 
         $results = $this->runChecks();
 
-        if (! $this->option('no-notification')) {
+        if (! $this->option('no-notification') && config('health.notifications.enabled', false)) {
```
Comment (kolaente) [inline src/Commands/RunHealthChecksCommand.php:32]: I'm not sure if this is the right place for this, but it fixes the problem for me.

**Comment by kolaente** [comment]
The test failure seems to be related to a change in space for a function, do the fixtures need adjustement?

**Comment by freekmurze** [comment]
Could you rebase against `main` to fix the tests?

**Comment by kolaente** [comment]
Done

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**